lib/presentation/extensions/client_extension.dart

```diff
@@ -27,4 +27,10 @@ extension ClientExtension on Client {
   String mxid(BuildContext context) => userID ?? L10n.of(context)!.user;
 
   String? get pusherNotificationClientIdentifier => userID?.sha256Hash;
+
+  bool isSingleAccountOnHomeserver(
+    Uri onHomeserver,
+  ) {
+    return isLogged() && homeserver == onHomeserver;
+  }
```

Comment:
not really understand what it is?

Reply:
It checks homeserver input already existed or not.

Reply:
change the name `onHomeserver`, it makes confused